config.md

```diff
@@ -12,7 +12,7 @@ Below is a detailed description of each field defined in the configuration forma
 
 * **`ociVersion`** (string, required) MUST be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OpenContainer specification with which the bundle complies.
 The OpenContainer spec follows semantic versioning and retains forward and backward compatibility within major versions.
-For example, if an implementation is compliant with version 1.0.1 of the spec, it is compatible with the complete 1.x series.
+For example, if a configuration is compliant with version 1.1 of this specification, it is compatible with all runtimes that support any 1.1 or later release of this specification, but it not compatible with a runtime that supports 1.0 and not 1.1.
```

Comment:
On Tue, Jul 26, 2016 at 03:00:34PM -0700, Aleksa Sarai wrote:
> > -For example, if an implementation is compliant with version 1.0.1 of the spec, it is compatible with the complete 1.x series.
> > +For example, if a configuration is compliant with version 1.1 of this specification, it is compatible with all runtimes that support any 1.1 or later release of this specification, but it not compatible with a runtime that supports 1.0 and not 1.1.
> 
> Doesn't this break our ability to have breaking changes on major
> releases? I prefer the 1.x series wording for this.

Neither line mentions major bumps at all.  If we want a major-bump
line, I suggest something like:

  Once a 2.0 spec is cut, a 1.x configuration will not be compatible
  runtimes that only support 2.x configurations (and nothing in the
  1.x line).